### PR TITLE
Use pop up to display error message instead of text.

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**
- Display an error message while creating a competition in an alert pop up instead of text.
- Add a pop up to confirm that the competition is created
Example:
image
image

### CODE REVIEW

**Review by texm — APPROVED** [review]
cool

**Review by roonil-wazlib — APPROVED** [review]
LGTM, might want Mercury's approval to but if it's blocking you then I'm approving now.

**Comment by lin8231** [comment]
No it's cool

**Review by lin8231 — APPROVED** [review]

**Pull request merged**
